### ISSUE TITLE
As a developer, I would like to be able to see latest changes from Nachet on the deployed instance

### ISSUE THREAD

**Issue by SonOfLope** [ai-cfia/howard#114]
The deployment is currently pointing to an old branch. I would like it to point to the main like Finesse does.

### Issue Description

Currently, the main branch of Nachet is not up to date with the latest deployment. The current branch `51-implementing-2-models` is pending review to able to be merged into `main`.

The actual issue with the deploy version is that it is not able to classify images. The return error `413 413 Request Entity Too Large` could be either an ingress error or a backend error.

This error started on Friday and was discovered by Taran around 4 pm.

Change between the moment the deployment work and the problem was reported.

We had discussion about increasing the content length for uploads 2 weeks ago.

* https://github.com/ai-cfia/nachet-backend/pull/66
* https://github.com/ai-cfia/nachet-backend/issues/64

### Test for investigation

- [x] Validate that the image source size is the same as the image size that was received by the backend.
  - Validate that the size is the same (Monday Morning).
- [x] Consult the log in ArgoCD while trying to classify an image.
  - Log doesn't update on both Nachet Container (Monday Morning)
 - [x] Validate that ingress value is the good one
 - [x] Update nachet `main` branch with latest change

**Comment by MaxenceGui** [comment]
### Report Issue

The default value for `client_max_body_size` in the ingex server is `1 MB` and corresponds to the size of the image I was trying to classify. When testing different points mentioned in the issue I got the following result:

- Image size in the backend was the same as the source image in the frontend
- When trying to classify my test image (~ `0.8 MB`) error `413` was returned but did not appear on any log in the backend.
- When trying to classy test images lower than `1 MB` (~ ` 0.1 MB` and ~ `0.5 MB`) the deployment was working and returned a result.

### Conclusion

Even if the value is set up in the [yaml file](https://github.com/ai-cfia/howard/blob/main/kubernetes/aks/apps/nachet/base/nachet-ingress.yaml) it seems the deployment doesn't follow that value of `200 MB` and still block object bigger than `1 MB`.

**Comment by MaxenceGui** [comment]
Still need to change the deploy branch to main (when main is ready)

**Comment by MaxenceGui** [comment]
> What are we waiting for before doing this ?

Still waiting for this PR to be merged:

* https://github.com/ai-cfia/nachet-backend/pull/55

Waiting on @RussellJimmies approval
